src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+automatically prompt and set focus on `required` fields with missing data, and you can use the 
+[`:required` pseudo-class](https://caniuse.com/?search=required) to highlight required fields. No 
+JavaScript required!
+
+Add an asterisk to the label for every required field, and add a note at the start of the form to 
+explain what the asterisk means.
+
+## Simplify checkout {: #checkout-forms}
+
+### Mind the mobile commerce gap! {: #m-commerce-gap}
+
+Imagine that your users have a **fatigue budget**. Use it up, and your users will leave. 
+
+You need to reduce friction and maintain focus, especially on mobile. Many sites get more **traffic** 
+on mobile but more **conversions** on desktop—a phenomenon known as the [mobile commerce gap](https://www.comscore.com/Insights/Presentations-and-Whitepapers/2017/Mobiles-Hierarchy-of-Needs). Customers may simply prefer to 
+complete a purchase on desktop, but lower mobile conversion rates are also a result of poor user 
+experience. Your job is to **minimize** lost conversions on mobile and **maximize** conversions 
+on desktop. [Research has shown](https://www.comscore.com/Insights/Presentations-and-Whitepapers/2017/Mobiles-Hierarchy-of-Needs) that there's a huge opportunity to provide a better mobile form experience. 
+
+Most of all, users are more likely to abandon forms that look long, complex and without a sense of 
+direction—especially on smaller screens and when users are distracted or in a rush. Ask for as 
+little data as possible.
+
+### Make guest checkout the default {: #guest-checkout}
+
+For an online store, the simplest way to reduce form friction is to make guest checkout the default. 
+Don't force users to create an account before making a purchase. Not allowing guest checkout is cited 
+as a major reason for shopping cart abandonment.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/shopping-cart-abandonment.png" alt="Reasons for shopping 
+  cart abandonment during checkout.">
+  <figcaption class="w-figcaption">From <a href="https://baymard.com/checkout-usability">baymard.com/checkout-usability</a></figcaption>
+</figure>
+
+You can offer account sign-up after checkout.  At that point, you already have most of the data you 
+need to set up an account, so account creation should be quick and easy for the user. 
+
+{% Aside 'gotchas' %}
+If you do offer sign-up after checkout, make sure that the purchase the user just made is linked to 
+their newly created account!
+{% endAside %}
+
+### Show checkout progress  {: #checkout-progress}
+
+You can make your checkout process feel less complex by showing progress and making it clear what 
+needs to be done next. The video below shows how UK retailer [johnlewis.com](https://www.johnlewis.com) 
+show checkout progress.
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/johnlewis.com.mp4" type="video/mp4">
```

Comment:
It would be nice to keep that last frame with the 123 progress line a while longer. I think you might be able to achieve the same effect with the `poster` attribute? If you end up redoing the video, I suggest also trimming the searching and sorting lawnmowers from the beginning.

Reply:
I've updated the video to trim the lawnmower search/sort. I've also added a freeze frame at the end for a few seconds. I've also added a poster attribute, but I'm not sure if that works.

I wonder if I should take off the `loop` (and maybe `autoplay` as well).

BTW this and the other videos will need to be uploaded to the web.dev Google Cloud Storage bucket. Let me know if I can help with that.